api/v2/latest/index.htm

```diff
@@ -1,7 +1,18 @@
 <html>
 <body>
 
-<p>At the present moment there isn't a "latest" version of the NGSIv2 spec. The specification is now closed and you can find the "stable" version <a href="../stable">here</a>.</p>
+<p>You probably arrived here looking for the NGSIv2 Specification document :)</p>
+
+<p>Instead of that, you should use the <a href="TBD">Orion API Specification</a>, which is based in the original NGSIv2 specification
```

Comment:
```suggestion
<p>Instead of that, you should use the <a href="https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md">Orion API Specification</a>, which is based in the original NGSIv2 specification
```